front/app/translations/admin/en.json

```diff
@@ -543,6 +543,10 @@
   "app.containers.Admin.pagesAndMenu.bottomInfoSaveButton": "Save Bottom Info Section",
   "app.containers.Admin.pagesAndMenu.homeTitle": "Home",
   "app.containers.Admin.pagesAndMenu.title": "Pages & Menu",
+  "app.containers.Admin.pagesAndMenu.topInfoContentEditorTitle": "Content",
+  "app.containers.Admin.pagesAndMenu.topInfoDescription": "The area at the top of the homepage can be customized with your own images, text and links.",
+  "app.containers.Admin.pagesAndMenu.topInfoPageTitle": "Top Info Section",
+  "app.containers.Admin.pagesAndMenu.topInfoSaveButton": "Save Top Info Section",
```

Comment:
To be reviewed afterwards, but normally we don't do capitalization like this. I think we rather do 'Top info section' and 'Save top info section' (or just 'Save').

Reply:
Yeah my instinct is to capitalize everything in buttons and such but our brand guidelines say otherwise. I'll also log a ticket to review all the final copy and include this as one of the changes